src/algebra/group/units.lean

```diff
@@ -39,6 +39,19 @@ variables [monoid α]
 
 @[to_additive] instance : has_coe (units α) α := ⟨val⟩
 
+@[to_additive] instance : has_inv (units α) := ⟨λ u, ⟨u.2, u.1, u.4, u.3⟩⟩
+
+/-- See Note [custom simps projection] -/
+@[to_additive /-" See Note [custom simps projection] "-/]
+def simps.coe (u : units α) : α := u
```

Comment:
I'm quite sure this one is figured out automatically, so you should be able to remove it.

I'm regranting the bors d+ :)

Reply:
It isn't automatic, without this definition it generates lemmas about `.val` instead of about `coe`.

Reply:
Hmm... I see.
It might only detect `has_coe_to_fun` instances automatically, not `has_coe` instances.